src/utils/formValidators.js

```diff
@@ -80,3 +82,59 @@ export const validateValueLength = (value, maxLength) => {
 export const validateValueLength1 = value => validateValueLength(value, 1);
 
 export const validateValueLength3 = value => validateValueLength(value, 3);
+
+/**
+ * Validate MARC path, quoted string and else condition. Match `910`, `910$a`, `"text"`,
+ * `910$a "text"`, `910$a; else "text"; else 910`,
+ * @param value
+ * @returns {null|*}
+ */
+export const validateMARCWithElse = value => {
+  const pattern = new RegExp(['^(("[^"]+")|([0-9]{3}(\\$[a-z])?))',
+    '(((\\s(?=(("[^"]+")|([0-9]{3}(\\$[a-z])?))))',
+    '((?<=\\s)(("[^"]+")|([0-9]{3}(\\$[a-z])?))))|',
+    '(((; else )(?=(("[^"]+")|([0-9]{3}(\\$[a-z])?))))',
+    '((?<=(; else ))(("[^"]+")|([0-9]{3}(\\$[a-z])?)))))*$'].join(''));
```

Comment:
I think this decision will make you happier, especially your eyes
```suggestion
  const template = '(("[^"]+")|([0-9]{3}(\\$[a-z])?))';
  const pattern = new RegExp([
    `^${template}`,
    `(((\\s(?=${template}))`,
    `((?<=\\s)${template}))|`,
    `(((; else )(?=${template}))`,
    `((?<=(; else ))${template})))*$`
    ].join(''));
```